kytos/core/log_manager.py

```diff
@@ -0,0 +1,85 @@
+"""Module with classes used to handle logs displayed by Kytos SDN Platform."""
+import logging
+from logging.handlers import SysLogHandler
+
+__all__ = ('LogManager')
+
+
+class LogManager:
+    """Class responsible to handle all logs send by kytos and the Napps."""
+
+    fmt = '%(asctime)s - %(levelname)s [%(name)s] (%(threadName)s) %(message)s'
+    handlers_enabled = []
+    loggers = {}
+
+    def __init__(self, mode='INFO'):
+        """Contructor of LogManager.
+
+        This method will create a LogManager instance with the mode given.
+
+        Args:
+            mode(string): Type of log that will be registered by kytos.
+        """
+        logging.basicConfig(format=self.fmt, level=getattr(logging, mode))
+
+    def new_logger(self, logger_name=None):
+        """Method used to instantiate a new logger.
+
+        Args:
+            logger_name(string): logger name that will be instantiated.
+        Returns:
+            logger(`logging.Logger`): instance of logging.Logger.
+        """
+        if logger_name not in self.loggers.keys():
+            logger = logging.getLogger(logger_name)
+            self.loggers[logger.name] = logger
+            self.__add_handlers_enabled_to_logger(logger)
+
+        return logger
```

Comment:
There is an `else` clause missing here. If `new_logger()` is called with a `logger_name` that already exists this method cannot return `logger` since it does not exist. Then, I think this method could be:
```
if logger_name not in self.loggers.keys():
    logger = logging.getLogger(logger_name)
    self.loggers[logger.name] = logger
    self.__add_handlers_enabled_to_logger(logger)
    return logger
else:
    return self.loggers.get(logger_name)
```